Added Kill HVT OBJ

Catshannon opened the pull request:
Added Kill HVT OBJ

Catshannon committed:
Added Kill HVT OBJ

Added Kill HVT OBJ

JohanLanges commented on SpecialOperations2.Altis/sos/mission/functions/fn_taskEliminateHVT.sqf:
```diff
@@ -9,5 +9,90 @@
+_task = ["EliminateHVT", position _hvt] call SOS_fnc_addTask;
+
+// Fail Task
+while {alive _hvt} do {
```
This is a future bug Cat. Because you call this function, a while loop can only iterate 10000 times. So this means the loop will fail when enough time has gone by. To be specific: 83 minutes ((10000 \* 0.5 seconds) / 60).
To prevent this bug from happening place the while loop in the [_hvt, _task] spawn at line 89. When you spawn something, a while loop will run indefinitely.

Catshannon committed:
Working better

need to rework this 

            if (alive _hvt) then {
                waitUntil {(_fpb distance _hvt) > 10000};
                //[_task] call SOS_fnc_failedTask; need to make.
                deleteVehicle _chopper;
                hideObjectGlobal _hvt;
            // failed task.
            } else {
                waitUntil {(_fpb distance _chopper) > 10000};
                deleteVehicle _chopper;
            };

Catshannon committed:
Update fn_taskEliminateHVT.sqf

Catshannon committed:
Update fn_taskEliminateHVT.sqf

Catshannon committed:
Failed Task

Failed Task

legman commented on SpecialOperations2.Altis/sos/CfgFunctions.hpp:
```diff
@@ -56,6 +56,7 @@ class SOS_Functions {
         class createTower {};
         class createZone {};
         class endMission {};
+        class failedTask {};
```
failTask

Catshannon committed:
Officer changes

Given revolver and set stances to match situation

legman commented:
we go over this later

Catshannon committed:
Fixed Bug

left variable out of the spawn

Catshannon committed:
failtask

Catshannon committed:
FailTask renamed

Catshannon committed:
FailedTask to FailTask

Catshannon committed:
_hvt setUnitRank "COLONEL";

_hvt setUnitRank "COLONEL"; was setUnitRank "COLONEL";

Catshannon committed:
Update fn_taskEliminateHVT.sqf

Catshannon committed:
Changed Pos system

Changed 
Pos system
Var names
added
Deletes crew and chopper

Catshannon committed:
Update fn_taskEliminateHVT.sqf

Catshannon committed:
fixed merge issue

Catshannon committed:
Merge issues

Catshannon committed:
Tabbing

Catshannon committed:
Update CfgTaskDescriptions.hpp

Tabbing and added (OPTIONAL) to task name
